index.html

```diff
@@ -5089,7 +5089,8 @@ <h3>Protocol Bindings</h3>
             <span class="rfc2119-assertion" id="bindings-requirements-scheme">
                 Every form in a WoT Thing Description MUST follow the requirements
                 of the <a>Protocol Binding</a> indicated by the URI scheme [[!RFC3986]] of its
-                <code>href</code> member.
+                <code>href</code> member as indicated by the Binding Templates specification at <a href="https://w3c.github.io/wot-binding-templates/#creating-a-new-protocol-binding">Creating a New Protocol Binding</a>.
```

Comment:
This language is difficult to understand, since it refers the entire binding specification. The implication on the URI scene should be clarified.

Reply:
It shouldn't be the case, for me the relative part is shown when I click on https://w3c.github.io/wot-binding-templates/#creating-a-new-protocol-binding